src/ng-package.schema.json

```diff
@@ -98,6 +98,11 @@
             "type": "string"
           },
           "default": ["dom", "es2015"]
+        },
+        "ignoreScripts": {
+          "description": "Remove scripts from package.json after build",
+          "type": "boolean",
+          "default": false
```

Comment:
Ok, from the npm blog post, I don't see a motivation why a library package would need to include lifecycle scripts. I would either

- make this option `true` by default, making it opt-in to publish lifecycle scripts
- or: make the `delete packageJson.scripts` behaviour turned on by default and non-overwriteable

Reply:
And maybe name it `keepLifecycleScripts`?

Reply:
sure.

Reply:
I kept that false, since I don't want to change the default behavior how ng-packagr works